src/ios.ts

```diff
@@ -178,6 +213,8 @@ export const iosApi = <RunTarget extends SupportedRunTarget<'ios'>>(
             } catch (err) {
                 throw new Error('Cannot connect using SSH.', { cause: err });
             }
+
+            await this._internal.setupEnvironment();
```

Comment:
You still have to update the README. As this only happens when the `ssh` capability is enabled and considering https://github.com/tweaselORG/appstraction/issues/59#issuecomment-1545720609, I guess you could just add a "These are installed automatically if…" note to the list of packages for now.

Reply:
Does `frida`depend on `root`, though? One could install `frida-server` on the device and use it without any need to access the terminal.

Reply:
Otherwise I think you are right. Do you want me to do this in this PR, or open an issue for it?

Reply:
> Does `frida`depend on `root`, though? One could install `frida-server` on the device and use it without any need to access the terminal.

I don't understand what you're saying. Unless you compile `frida-server` into an app, you can only use it on a jailbroken device, so it does require root?

Reply:
> Otherwise I think you are right. Do you want me to do this in this PR, or open an issue for it?

Issue is fine for now.

Reply:
> I don't understand what you're saying. Unless you compile `frida-server` into an app, you can only use it on a jailbroken device, so it does require root?

Yes, but you don’t need to necessarily give appstraction root access, if you installed it yourself.

Reply:
Hm, I guess you could view it that way.